### PR TITLE
Feat/weather

### PR DESCRIPTION
**Pull request by Speedfan94**

### CODE REVIEW

**Review by weisske — APPROVED** [review]
Nice fix

**Review by mulko93 — APPROVED** [review]
looks good. changes can be merged

File: nextbike/prediction/math_prepare_feature.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ def create_new_features(p_X):
     # TODO Weather Data adding implement here, by triggering
 
     p_X = prediction.add_weather_data.add_weather(p_X)
+    # p_X["Rain"] = np.square(p_X["Hour_start"])
```
Comment (weisske) [inline nextbike/prediction/math_prepare_feature.py:37]: Please square rain as rain_squared :dagger:

**Review by MaxStroh — APPROVED** [review]
Looks fine :)

**Review by weisske — APPROVED** [review]

**Pull request merged**